### PR TITLE
Include Ruby STI scripts

### PR DESCRIPTION
**Pull request by strzibny**
This is a PR that should fix definition for Ruby scls and add STI scripts to Rails scls. Review needed.

@mfojtik @jhadvig @hhorak

### CODE REVIEW

File: rhscl.yaml [hunk 1]
```diff
@@ -267,7 +273,13 @@ containers:
     add:
         - [python-common/sti, sti, /usr/share/]
   ror40:
-    install: ror40 @devel_libs
+    install: ror40 nodejs010 @devel_libs
+    enable: [ror40, nodejs010]
+    add:
+        - [ruby-common/sti, sti, /usr/share/]
```
Comment (mfojtik) [inline rhscl.yaml:275]: maybe make /usr/share/sti ENV variable? ($STI_SCRIPTS_DIR?)

File: ruby-common/sti/bin/usage [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+DISTRO=`cat /etc/*-release | grep ^ID= | grep -Po '".*?"' | tr -d '"'`
+
+cat <<EOF
+This is a STI ruby-2.0 ${DISTRO} base image:
+To use it, install STI: https://github.com/openshift/source-to-image
+
+Sample invocation:
+
+sti build https://github.com/openshift/sti-ruby.git --context-dir=2.0/test/puma-test-app/ openshift/ruby-20-${DISTRO}7 ruby-sample-app
```
Comment (mfojtik) [inline ruby-common/sti/bin/usage:11]: you will have to change this.

**Comment by strzibny** [comment]
Paths have to be altered, will do based on https://github.com/sclorg/rhscl2dockerfile/issues/29 resolution.

File: rhscl.yaml (hunk 1 above)
Comment (strzibny) [inline rhscl.yaml:275]: @hhorak ideas on $STI_SCRIPTS_DIR?

**Comment by hhorak** [comment]
Sorry, this repos was made deprecated. If the issue is still in place in some of the container under https://github.com/sclorg?q=-container, then, please, report an issue for the specific container image.

**Pull request closed without merging**